tests/h/search/query_test.py

```diff
@@ -50,27 +50,27 @@ def test_it_uses_old_filter_syntax_for_es1(self):
         # TODO - Remove this test once ES 1 support is dropped.
 
         builder = Builder(es_version=(1, 7, 0))
-        filter = AuthorityFilter("default")
+        filter_ = AuthorityFilter("default")
```

Comment:
`filter` is a builtin rather than a reserved keyword (see https://docs.python.org/3/reference/lexical_analysis.html#keywords), so using it is an identifier is legal. I know that some syntax plugins for popular editors colorize builtins which could confuse the reader, so I'm OK with this change. Could you update the commit message though to describe what `filter` is more accurately?